eclipse-ankaios/README.md

```diff
@@ -0,0 +1,189 @@
+
+## Links
+
+- [Ankaios docs](https://eclipse-ankaios.github.io/ankaios/0.2/)
+- [Ankaios quickstart](https://eclipse-ankaios.github.io/ankaios/0.2/usage/quickstart/)
+- [Podman](https://docs.podman.io/en/v4.6.1/)
+- [What are devcontainers?](https://containers.dev/)
+
+## Prerequisites
+
+- Docker [Installation instructions](https://docs.docker.com/get-docker/)
+
+## Development environment
+
+The following is provided inside the devcontainer:
+
+- Ankaios executables (`ank-server`, `ank-agent` and `ank`)
+
+- Podman 4.6.2
+
+- Pre-configured Ankaios startup config [startupState.yaml](./config/startupState.yaml)
+
+- Automation scripts for starting and stopping all workloads of the challenge:
+    - run_blueprint.sh
+    - shutdown_blueprint.sh
+
+- Exposed port:
+    - 25551: for optionally using the Ankaios CLI outside of the devcontainer
+
+- [Ankaios Control Interface dependencies](#ankaios-control-interface-dependencies)
+
+
+All services are running in the host network meaning those can be accessed by `localhost:<port>`. We recommend that you set the network mode to host for all your developed workloads as well.
+
+## Ankaios Control Interface dependencies
+
+The devcontainer includes also dependencies for developing applications using the [Ankaios Control Interface](https://eclipse-ankaios.github.io/ankaios/0.2/reference/control-interface/):
+
+- protobuf compiler
+- grpcurl
+- Ankaios protobuf file (under `/usr/local/lib/ankaios/ankaios.proto`)
+
+Those dependencies are needed for use-cases in which your app needs to use the [Ankaios Control Interface](https://eclipse-ankaios.github.io/ankaios/0.2/reference/control-interface/) to be able to communicate with the Ankaios cluster via the API. An example use-case would be to write a workload that shall request Ankaios to dynamically start another workload. You can find example workload applications written in various programming languages inside the Eclipse Ankaios repository [here](https://github.com/eclipse-ankaios/ankaios/tree/v0.2.0/examples).
+
+## Run devcontainer with VSCode
+
+### Prerequisites
+- [Remote Development](https://marketplace.visualstudio.com/items?itemName=ms-vscode-remote.vscode-remote-extensionpack) extension installed in VSCode
+
+Open the subfolder containing this README file in VSCode:
+
+```shell
+code .
+```
+
+VSCode detects automatically that a `.devcontainer` folder exists inside this subfolder.
+Please confirm the dialog to reopen VSCode inside the devcontainer.
+Afterwards, open a new terminal inside the devcontainer in VSCode.
+
+## Run devcontainer without VSCode
+
+Navigate to the subfolder containing this README file and run the following command to build the devcontainer image:
+
+```shell
+docker build -t custom-ankaios-dev:0.1 --target dev -f .devcontainer/Dockerfile .
+```
+
+Todo! Fix mountpoints, when new folder structure of Microsoft is available..
```

Comment:
I guess you want to fix this Todo! with a later PR? Or did this slip through on your end?